comments/views.py

```diff
@@ -187,11 +185,11 @@ def post_comment(request, send_signal=True, extra=None):
         raise FailSafelyException("You are not editing the most recent version of this comment. Please refresh your page and try again.")
 
     # Everything has checked out, so we save the new version and return the appropriate response
-    extra = {**(extra or {})}
-    extra["parent_object"] = parent_object
-    version_form, new_version = create_new_version(request, comment, extra=extra)
+    kwargs = {**(kwargs or {})}
```

Comment:
the default value for kwargs is already a dictionary, so you should be able to remove this